Components/Queues/Drivers/PhpAmqpLibDriver.php

```diff
@@ -0,0 +1,385 @@
+<?php
+
+
+namespace Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers;
+
+use PhpAmqpLib\Channel\AMQPChannel;
+use PhpAmqpLib\Connection\AMQPStreamConnection;
+use PhpAmqpLib\Exception\AMQPRuntimeException;
+use PhpAmqpLib\Message\AMQPMessage;
+use PhpAmqpLib\Wire\AMQPTable;
+use Smartbox\CoreBundle\Type\SerializableInterface;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\PhpAmqpLibQueueManager;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\QueueManager;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\QueueMessage;
+use Smartbox\Integration\FrameworkBundle\Components\Queues\QueueMessageInterface;
+use Smartbox\Integration\FrameworkBundle\Core\Messages\Context;
+use Smartbox\Integration\FrameworkBundle\DependencyInjection\Traits\UsesSerializer;
+use Smartbox\Integration\FrameworkBundle\Exceptions\Handler\UsesExceptionHandlerTrait;
+use Smartbox\Integration\FrameworkBundle\Service;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
+
+/**
+ * Class PhpAmqpLibDriver
+ * @package Smartbox\Integration\FrameworkBundle\Components\Queues\Drivers
+ */
+class PhpAmqpLibDriver extends Service implements PurgeableQueueDriverInterface
+{
+    use UsesSerializer;
+    use UsesExceptionHandlerTrait;
+    use ContainerAwareTrait;
```

Comment:
Which is the reason this driver needs to be aware of the container?

Reply:
No needs. I removed the use of the Trait.